### PR TITLE
Showing result table with new results

### PR DESCRIPTION
**Pull request by smiechuskce**
Users results will be show at Results page in tab layout - each tab per single week.

## Motivation and context
#214 #155 

## How has this been tested?
Prepare table Result with scores (empty fields for inactive weeks/final score have to have NULL value). Run app, log in, open Results page.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/HomeService.cs [hunk 1]
```diff
@@ -55,12 +55,38 @@ public List<TestWithAnswerListDto> GetTestsWithUserAnswers()
             return testWithAnswersDtoList;
         }
 
-        public List<TestResultDto> GetAllTestResults()
+        public Dictionary<int, List<TestResultDto>> GetAllTestResults()
         {
-            var testResultList = this._homeRepository.GetAllTestResults();
-            var testResultsDtoList = this._mapper.Map<List<TestResultDto>>(testResultList);
+            var testResultDictionary = new Dictionary<int, List<TestResultDto>>();
```
[the remark this answers is not in the document]
Comment (smiechuskce) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/HomeService.cs]: I don't think it is necessary. Splitting does not make this code more elegant or readable. And all of those proposed methods will be as near as makes no difference than existing code - I think that variable names says clearly what they do. We should instead change entirely the approach of getting users' results.

[the remark this answers is not in the document]
Comment (WTobor) [reply src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/HomeService.cs]: ok, added new task #255

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/HomeService.cs [hunk 2]
```diff
@@ -120,5 +120,43 @@ public string PrepareUserEmailForRODO(string email)
...
+                        dateFrom = new DateTimeOffset(DateTime.Now.Year, 12, 8, 20, 0, 0, TimeSpan.Zero);
+                        dateTo = new DateTimeOffset(DateTime.Now.Year, 12, 15, 20, 0, 0, TimeSpan.Zero);
+                        break;
+                    case 3:
+                        dateFrom = new DateTimeOffset(DateTime.Now.Year, 12, 15, 20, 0, 0, TimeSpan.Zero);
+                        dateTo = new DateTimeOffset(DateTime.Now.Year, 12, 22, 20, 0, 0, TimeSpan.Zero);
+                        break;
+                    default:
+                        dateFrom = new DateTimeOffset(DateTime.Now.Year, 12, 1, 20, 0, 0, TimeSpan.Zero);
+                        dateTo = new DateTimeOffset(DateTime.Now.Year, 12, 25, 20, 0, 0, TimeSpan.Zero);
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Services/HomeService.cs]: 25.12, 00:00, not 20:00

File: src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Repository/HomeRepository.cs [hunk 3]
```diff
@@ -1,10 +1,13 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Linq;
+using System.Linq.Expressions;
```
Comment (WTobor) [inline src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Repository/HomeRepository.cs]: please add migration to set NULL by default for all Points and Places in Results table

**Pull request merged**